bootstrap/kustomization.yaml

```diff
@@ -0,0 +1,16 @@
+---
+namespace: argo-cd 
```

Comment:
I think this will need to be removed and controlled from the runtime config when we pull in this as a remote base.